### PR TITLE
Create Bubble sort in Rust (issue  #2143)

### LINKED ISSUE #2143: Bubble Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart
- [x] Go
- [x] Java
- [x] Julia
- [x] JS
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @m-code12
- [x] Shell
- [ ] TypeScript - @manildayma

### PR DESCRIPTION
**Pull request by aryangulati**
Fixes #2143

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- added bubble sort 
- 

#### Languages Used:
- in Rust language
- 

#### Files Added:
- Bubble_Sort/Bubble_Sort.rs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Bubble_Sort/Bubble_Sort.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            if compare_fn(target[j], target[j+1]) > 0  {
+                temp = target[j+1];
+                target[j+1] = target[j];
+                target[j] = temp;
+            }
+        }
+    }
+
+    target
+}
```
Comment (aryangulati) [inline Bubble_Sort/Bubble_Sort.rs]: sample input: 2 3 4 5 6 7 1
output :1 2 3 4 5 6 7

File: Bubble_Sort/Bubble_Sort.rs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+fn bubble_sort(numbers: &Vec<i64>, compare_fn: |i64, i64| -> i64) -> Vec<i64> {
```
Comment (jainaman224) [inline Bubble_Sort/Bubble_Sort.rs]: Please take input from user. It is just a function to sort.

**Comment by jainaman224** [comment]
@aryangulati Are you still working on it?

**Comment by somya-kapoor** [comment]
@aryangulati Are you working on it?

**Comment by aryangulati** [comment]
Yes I am working

**Comment by aryangulati** [comment]
Yes

**Comment by somya-kapoor** [comment]
@aryangulati Are you working on it?

**Comment by aryangulati** [comment]
No

**Pull request closed without merging**